packages/terra-application-layout/docs/utils.md

```diff
@@ -0,0 +1,99 @@
+# Terra Application Layout
```

Comment:
Add utilities to the header.

Reply:
It matches the other titles so that we have a consistent header among the other site pages.